tests/platform_tests/sfp/test_sfputil.py

```diff
@@ -331,13 +358,25 @@ def test_check_sfputil_presence(duthosts, enum_rand_one_per_hwsku_frontend_hostn
     # For vs testbed, we will get expected Error code `ERROR_CHASSIS_LOAD = 2` here.
     if duthost.facts["asic_type"] == "vs" and sfp_presence['rc'] == 2:
         return
-    assert sfp_presence['rc'] == 0, "Run command '{}' failed".format(cmd_sfp_presence)
+    assert sfp_presence['rc'] == 0, (
+        "Run command '{}' failed with return code {}. "
+        "This means the 'sfputil show presence' command did not execute successfully on the DUT. "
+        "Please check the command output, DUT logs, and platform status for troubleshooting."
+    ).format(cmd_sfp_presence, sfp_presence.get('rc', 'N/A'))
 
     parsed_presence = parse_output(sfp_presence["stdout_lines"][2:])
     for intf in dev_conn:
-        if intf not in xcvr_skip_list[duthost.hostname]:
-            assert intf in parsed_presence, "Interface is not in output of '{}'".format(cmd_sfp_presence)
-            assert parsed_presence[intf] == "Present", "Interface presence is not 'Present'"
+        if intf not in xcvr_skip_list.get(duthost.hostname, []):
+            assert intf in parsed_presence, (
+                "Interface '{}' is not in output of '{}'. "
+                "Parsed presence output: {}".format(intf, cmd_sfp_presence, parsed_presence)
+            )
+            assert parsed_presence[intf] == "Present", (
+                "Interface '{}' presence is not 'Present'. "
+                "This means the SFP module is not detected as present on this interface. "
+                "Please verify the SFP module is properly inserted and operational, check compatibility, "
+                "run 'sfputil show presence' manually, and review DUT logs for related errors."
+            ).format(intf)
```

Comment:
Please remove these 3 indication lines. Just print out the reason is enough, don't need to explain or tell the user how to do debug it. "This means is not necessary at all"